### ISSUE TITLE
“Ask for help with this question” disappears while hovered

### ISSUE THREAD

**Issue by zoul** [theodi/open-data-certificate#1204]
The “ask for help with this question” link sometimes disappears when I try to click it:

![a screen recording of the issue](https://cloud.githubusercontent.com/assets/95329/8748893/d1c91ef4-2c9d-11e5-9134-5b8ed5ce09e9.gif)

There are already some closed issues related to this problem, but I have found none open, so I’m creating a new one. Looks like the problem could be caused by the orange hint below the input text field?

**Comment by zoul** [comment]
This is the Czech version, but the English one behaves the same way (the `copyrightURL` question is displayed here). I briefly tried to debug the issue, but there’s quite a lot going on. Given how much information is now displayed for each question (question text itself, certificate level hints, certificate level suggestions, help text), the styling and interplay are getting a bit complex. Wouldn’t it be nice to have a static prototype containing two or three questions to test and debug the styling on?

**Comment by danielgavrilov** [comment]
Yes, this is because the container of the orange hint appears above the help message. The bug can be seen by resizing window width below `768px` and mousing over the field mentioned. 

One way to fix this is to set `z-index` of `.question-meta` probably above `5` (which seems to be the z-index of `<select>` elements), but below `10` (which is z-index of the sticky header). I'm not very familiar with the code, so I'm not completely sure whether this would break anything?
